src/containers/auth/register/Register.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button } from 'react-bootstrap';
+
```

Comment:
Lets remove the additional new line here

Reply:
Fixed